Automattic/woocommerce-payments
# Add pagination and summary to Disputes table

Fixes #3386 


#### Changes proposed in this Pull Request


Adds summary information and pagination controls to the Disputes table. 

<img width="1562" alt="Screen Shot 2021-12-26 at 19 20 18" src="https://user-images.githubusercontent.com/1553182/147416887-a66a8395-472d-4518-a5c8-1e6466024e84.png">







#### Testing instructions





1. Add a number of orders with disputes greater than the rows per page defined for the disputes table. This number is set to 25. Alternatively, you can use the `per_page` query parameter to set a smaller number i.e. `http://localhost:8082/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Fdisputes&paged=1&per_page=10`
2. You can use [Stripe's test cards to pay for the orders](https://stripe.com/docs/testing#disputes). Using one of this cards will generate a dispute after the order is paid.
3. Note that this PR relies on the pagination functionality developed [here](1415-gh-automattic/woocommerce-payments-server)
4. Follow instructions from 1408-gh-Automattic/woocommerce-payments-server to have disputes saved in the cache
5. Observe that the pagination controls appear at the bottom of the table.
6. Check that the pagination controls work as expected (navigate to next/ prev page, go to page number, select rows per page)
7. Observe that the total number of Disputes is displayed at the bottom of the table

-------------------


**Post merge**
